lib/kvs/kvs_rparams.c

```diff
@@ -444,43 +402,43 @@ static const struct param_spec pspecs[] = {
     },
     {
         .ps_name = "cn_mcache_kra_params",
-        .ps_description = "kblock readahead [pct][lev1][lev0]",
+        .ps_description = "kblock readahead [willneed]",
         .ps_flags = PARAM_FLAG_EXPERIMENTAL,
-        .ps_type = PARAM_TYPE_U64,
+        .ps_type = PARAM_TYPE_U8,
```

Comment:
Can be a bool?

Reply:
Could, but I don't know if this is the end of it, we should know better in a few weeks...